godtools/AppDelegate.swift

```diff
@@ -80,9 +76,22 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         // Initializes the importer so the resources directory can be created.
         _ = TranslationZipImporter.shared
```

Comment:
That changes the declaration from `let` to `var` and makes `shared` and optional. Unless there's a way to have the setup `class` function call a private init method? Have you done something like the before? I'd strongly prefer the singleton not be an optional that has to be unwrapped each time.

````swift
    var shared: TranslationZipImporter?
    
    class func setup() {
        shared = TranslationZipImporter()
    }
````